extensions/migadu/package.json

```diff
@@ -59,17 +59,17 @@
     }
   ],
   "dependencies": {
-    "@raycast/api": "^1.50.2",
-    "@raycast/utils": "^1.5.2",
+    "@raycast/api": "^1.80.0",
+    "@raycast/utils": "^1.16.3",
     "node-fetch": "^3.3.1"
   },
   "devDependencies": {
-    "@raycast/eslint-config": "1.0.5",
-    "@types/node": "18.8.3",
-    "@types/react": "18.0.9",
+    "@raycast/eslint-config": "^1.0.8",
+    "@types/node": "^20.8.10",
+    "@types/react": "^18.3.3",
     "eslint": "^7.32.0",
     "prettier": "^2.5.1",
```

Comment:
Btw there is a moderate severity due to these dependencies, can you please execute `npm i eslint@8 prettier@3 -D`?